libyul/backends/wasm/BinaryTransform.cpp

```diff
@@ -566,11 +616,11 @@ bytes BinaryTransform::globalSection()
 	return makeSection(Section::GLOBAL, std::move(result));
 }
 
-bytes BinaryTransform::exportSection()
+bytes BinaryTransform::exportSection(std::map<std::string, size_t> const& _registeredFunctions)
```

Comment:
```suggestion
bytes BinaryTransform::exportSection(map<string, size_t> const& _registeredFunctions)
```
(and other places)

Reply:
Done. I removed all prefixes from parameter declarations and from function calls like `std::get()` or `std::move()`. `std::visit()` still needs the prefix because there's a function with that name in the class.